bentoml/pytorch_lightning.py

```diff
@@ -18,18 +19,27 @@
 """
 
 if TYPE_CHECKING:  # pragma: no cover
+    import pytorch_lightning as pl
+
```

Comment:
considering this module is bentoml.pytorch_lightning, I think it's ok to import pytorch_lightning directly?

Reply:
funny thing is we don't even use pytorch_lightning for this. Mostly just served for typing